### PR TITLE
kola test to check the network device info

### PR DESCRIPTION
**Pull request by aaradhak**
Ref: https://github.com/coreos/fedora-coreos-tracker/issues/1153
BZ : https://bugzilla.redhat.com/show_bug.cgi?id=2080043

### CODE REVIEW

File: tests/kola/networking/network-device-info [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# kola: { "exclusive": false }
+
+set -xeuo pipefail
+
+. $KOLA_EXT_DATA/commonlib.sh
+
+# See - https://github.com/coreos/fedora-coreos-tracker/issues/1153
+
+if [ ! -f "/etc/issue.d/22*.issue" ]; then
```
Comment (dustymabe) [inline tests/kola/networking/network-device-info]:
```suggestion
if [ ! -f /etc/issue.d/22*.issue ]; then
```

If you quote this it won't expand the `*`.

File: tests/kola/networking/network-device-info [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# kola: { "exclusive": false }
+
+set -xeuo pipefail
+
+. $KOLA_EXT_DATA/commonlib.sh
+
+# See - https://github.com/coreos/fedora-coreos-tracker/issues/1153
```
Comment (dustymabe) [inline tests/kola/networking/network-device-info]: Let's move this to the top and add a comment explaining what the test does.

```
This test makes sure CLHM wrote a snippet for the NIC IP info. See https://github.com/coreos/fedora-coreos-tracker/issues/1153
```

File: tests/kola/networking/network-device-info [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# kola: { "exclusive": false }
```
Comment (dustymabe) [inline tests/kola/networking/network-device-info]: We can now do this in yaml (and add comments!)

```suggestion
## kola:
##   # This is a read-only test that can be run with other tests.
##   exclusive: false
```

**Review by dustymabe — COMMENTED** [review]
Since this is testing CLHM how about we move the test into `tests/kola/clhm/network-device-info ?

**Comment by dustymabe** [comment]
we'll need to add a denylist entry for this test for `testing-devel`, `testing`, and `stable`. until the selinux fix makes it in. Just set the test to snooze for 3 weeks and we'll re-evaluate after that point in time.

**Comment by aaradhak** [comment]
> we'll need to add a denylist entry for this test for `testing-devel`, `testing`, and `stable`. until the selinux fix makes it in. Just set the test to snooze for 3 weeks and we'll re-evaluate after that point in time.

is a separate PR required for the denylist entry?

**Comment by dustymabe** [comment]
> is a separate PR required for the denylist entry?

no, but I would do it in a separate commit

File: kola-denylist.yaml [hunk 4]
```diff
@@ -37,3 +37,10 @@
   streams:
   - branched
   - rawhide
+pattern: network-device-info
```
Comment (dustymabe) [inline kola-denylist.yaml]:
```suggestion
- pattern: ext.config.clhm.network-device-info
```

File: tests/kola/clhm/network-device-info [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+## kola:
+	##   # This is a read-only test that can be run with other tests.
+	##   exclusive: false
```
Comment (dustymabe) [inline tests/kola/clhm/network-device-info]:
```suggestion
## kola:
##   # This is a read-only test that can be run with other tests.
##   exclusive: false
```

File: kola-denylist.yaml [hunk 6]
```diff
@@ -37,3 +37,10 @@
   streams:
   - branched
   - rawhide
+pattern: ext.config.clhm.network-device-info
```
Comment (dustymabe) [inline kola-denylist.yaml]:
```suggestion
- pattern: ext.config.clhm.network-device-info
```

**Comment by dustymabe** [comment]
I think you also need to create the `data/commonlib.sh` symlink.

```
$ mkdir tests/kola/clhm/data
$ ln -s ../../data/commonlib.sh tests/kola/clhm/data/commonlib.sh
$ git add tests/kola/clhm/data/commonlib.sh
```

**Comment by aaradhak** [comment]
> ln -s ../../data/commonlib.sh tests/kola/clhm/data/commonlib.sh

I forgot about this part. I ll add it.

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**